### PR TITLE
Added support for iOS 15 property interruption-level

### PR DESCRIPTION
**Pull request by simonmaddox**
A copy of the PR on https://github.com/Netatmo/node-apn/pull/6

See more [here](https://developer.apple.com/videos/play/wwdc2021/10091/), [here](https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel?changes=la) and [here](https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/generating_a_remote_notification).

### CODE REVIEW

**Review by cbaker6 — APPROVED** [review]

**Comment by cbaker6** [comment]
@davimacedo are you able to look this over?

File: lib/notification/apsProperties.js [hunk 1]
```diff
@@ -122,6 +122,12 @@ module.exports = {
     }
   },
 
+  set interruptionLevel(value) {
+    if(typeof value === "string" || value === undefined) {
```
Comment (TysonAndre) [inline lib/notification/apsProperties.js:126]:
```suggestion
    if (typeof value === "string" || value === undefined) {
```

nit: use the same spacing used elsewhere

File: test/notification/apsProperties.js [hunk 2]
```diff
@@ -711,6 +711,32 @@ describe("Notification", function() {
...
+        note.interruptionLevel = "the-interruption-level";
+        note.interruptionLevel = undefined;
+
+        expect(compiledOutput()).to.not.have.nested.property("aps.interruption\-level");
+      });
+
+      describe("setInterruptionLevel", function () {
+        it("is chainable", function () {
+          expect(note.setInterruptionLevel("the-interruption-level")).to.equal(note);
+          expect(compiledOutput()).to.have.nested.property("aps.interruption\-level", "the-interruption-level");
```
Comment (TysonAndre) [inline test/notification/apsProperties.js:735]:
```suggestion
          expect(compiledOutput()).to.have.nested.property("aps.interruption-level", "the-interruption-level");
```

nit: unnecessary escaping

**Review by TysonAndre — COMMENTED** [review]
lgtm other than code style nits

**Comment by davimacedo** [comment]
@TysonAndre since the code nits are existing one, I'm going to merge this one. Maybe we should install lint/prettier to this repo. What do you think?

**Pull request merged**